### DISCUSSION TITLE
Error in loading marker on Worker 404

### DISCUSSION THREAD

**Discussion by PeraDev7** [AR-js-org/AR.js#373] [Help]
**Do you want to request a *feature* or report a *bug*?**
BUG

**What is the current behavior?**
Error in loading marker on Worker 404

**If the current behavior is a bug, please provide the steps to reproduce.**
Open website 
https://acapulcobeach.it/ar/
In all browsers i get this error. Image files made with node js version of NFT MARKER CREATOR.
Both files are on the linked directory.

Github Project : 
https://github.com/PeraDev7/TagShirt-AR

**What is the expected behavior?**
Run the app

**Comment by kalwalt** [comment]
Hi @PeraDev7 this is not a bug you give the wrong path to the nft marker:
`url="assets/logo/"`
ref here: https://github.com/PeraDev7/TagShirt-AR/blob/db6e84742794a652926a0f7c1fa59013cbdfe01f/index.html#L45
according to your assets and files instead it should be:
`url="assets/logo/logo"`

**Reply by Aatmik-Coder** [reply]
![error](https://user-images.githubusercontent.com/75309042/202371229-d3fe5d97-f537-4783-9f5d-05a40873ecd5.jpg)
 I am getting the same error when I deploy it, in local it is working as expected.
why? 
please let me know ASAP.
Thank you in advance.

**Reply by mbugua70** [reply]
Hello sir, did you get help on this issue
